Mathlib/Algebra/Group/WithOne/Defs.lean

```diff
@@ -239,11 +239,13 @@ theorem coe_mul {α : Type u} [Mul α] {a b : α} : ((a * b : α) : WithZero α)
 -- prove it because we've just proved we're in MulZeroClass.
 theorem zero_mul {α : Type u} [Mul α] (a : WithZero α) : 0 * a = 0 :=
   rfl
-#align with_zero.zero_mul WithZero.zero_mul
+-- Porting note: This lemma and the next one
```

Comment:
Have you looked into why that is? Otherwise maybe leave it for now?

Reply:
Well `with_zero.zero_mul` definitely doesn't exist. So it doesn't make a lot of sense for it to have an `#align`, right? regardless of what else is going on?
But we could just leave an erroneous align, as well (it's not like there will be none, after this PR).